spec/git-version-spec.cr

```diff
@@ -505,4 +505,22 @@ describe GitVersion do
       tmp.cleanup
     end
   end
+
+  it "should increase the patch release if the patch command exit with 0" do
+    tmp = InTmp.new
+
+    begin
+      git = GitVersion::Git.new("dev", "master", "if [ \"1.0.0\" = \"<version>\" ]; then exit 0; else exit 1; fi", tmp.@tmpdir)
+
+      tmp.exec %(git init)
+      tmp.exec %(git checkout -b master)
+      tmp.exec %(git commit --no-gpg-sign --allow-empty --no-gpg-sign -m "breaking: 1")
+
+      version = git.get_version
+      hash = git.current_commit_hash
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Useless assignment to variable `hash`](https://app.codacy.com/gh/codacy/git-version/pullRequest?prid=4909832)